### ISSUE TITLE
Update tokens and file structure - WIP

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-equitable-development-tool#213]
Requested OSE changes:

- Change "pct_moe" to "pctmoe"
- Add "_count" to indicate a value is for an estimate, instead of no token
- Add "tpop" to indicate a value is for the total pop and not an specific race/ethnic group, instead of no token
- Organize files such that the year that a column of data corresponds to is consistently communicated by either the file name or by column names.

### Level of effort

Change values in the following files:

- [aggregate/quality_of_life/diabetes_self_report.py](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/aggregate/quality_of_life/diabetes_self_report.py)
- [aggregate/quality_of_life/access_transit_car.py](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/aggregate/quality_of_life/access_transit_car.py)
- [notebooks/household_income_bands_checks.ipynb](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/notebooks/household_income_bands_checks.ipynb)
- [aggregate/quality_of_life/access_to_broadband.py](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/aggregate/quality_of_life/access_to_broadband.py)
- [aggregate/housing_security/units_affordable.py](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/aggregate/housing_security/units_affordable.py)
- [aggregate/PUMS/pums_2000_economics.py](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/aggregate/PUMS/pums_2000_economics.py)
- [aggregate/PUMS/pums_2000_demographics.py](https://github.com/NYCPlanning/db-equitable-development-tool/blob/855ad5011210ff1cedbb2598c2bd6f25e8a8f424/aggregate/PUMS/pums_2000_demographics.py)

Reorganize files to consistently communicate years

Rules:
- If a data point is available for 2 or more time points consistently communicate year.
- If there is a single data point for a single point in time or cumulative across a timespan, then the year is not included.

Option 1: Consistently communicate year in field name
Changes:
- Demographic
9 files become 3 files, with over 600 fields each.  Add decennial census year or ACS year token to every field.
- Economic
9 files become 3 files, with nearly 3000 fields each.  Add decennial census year or ACS year token to every field.

Option 2: Consistently communicate year in file name
- QOL and Opportunity
Split out health outcome indicators into separate files for each of the following time periods:
- 2000
- 2010
- 2019
- 2000 - 2004
- 2010 - 2014
- 2015 - 2019
Split out public safety indicators into separate files for each of the following time periods:
- 2010 - 2014
- 2016 - 2020
Split out indicators coming from PUMS data for car commute into 
- 2008 - 2012
- 2015 - 2019 (add to health)

- Housing security and quality
Split out indicators into the following groups
- 1 time period
- 2000
- 2008-2012
- 2015-2019
- 2018
- 2020

**Comment by mbh329** [comment]
I think the path of least resistance is probably just adding the year to the field name but would that be a cumbersome lift on the data dictionary side?

**Comment by AmandaDoyle** [comment]
> I think the path of least resistance is probably just adding the year to the field name but would that be a cumbersome lift on the data dictionary side?

Noted.  I'm not worried about the data dictionary.  I am concerned about shoving 3000+ fields into a single file though.

**Comment by SashaWeinstein** [comment]
I agree with what's been voiced that avoiding 3k column tables is best if possible
